WinUIGallery/Controls/PageHeader.xaml.cs

```diff
@@ -64,6 +64,7 @@ private void OnCopyLinkButtonClick(object sender, RoutedEventArgs e)
         public void OnThemeButtonClick(object sender, RoutedEventArgs e)
         {
             ToggleThemeAction?.Invoke();
+            UIHelper.AnnounceActionForAccessibility(ThemeButton, "Theme Changed.", "ThemeChangedSuccessNotificationId");
```

Comment:
nit: spelling

Reply:
What did I spell wrong...? Is it the Capital C?

Reply:
Yes though users of the app will not notice this